### PR TITLE
workflow(all): optimize workflow

### PR DESCRIPTION
**Pull request by DevHaoZi**
## 📑 Description
对工作流做了一些改进如下：

1. 测试ChatGPT自动Review PR
2. 改进了提交语法，遵从：[github.com/carloscuesta/gitmoji](https://github.com/carloscuesta/gitmoji)，简中版本：[gitmoji.js.org](https://gitmoji.js.org)
3. 测试自动更新贡献者
4. 移除Welcome的Action，它工作不太正常。

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information
先在这边测试，无误后再提交到goravel主仓库。

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -8,14 +8,13 @@ permissions:
   contents: read
 jobs:
   golangci:
```
Comment (hwbrzzl) [inline .github/workflows/lint.yml:10]:
```suggestion
  lint:
```

**Review by hwbrzzl — COMMENTED** [review]
Interesting!

**Comment by DevHaoZi** [comment]
突然想起来master有写保护，这样的话自动更新贡献者应该是用不了了，我等下去掉。

File: .github/workflows/check-pr-title.yml
Comment (hwbrzzl) [inline .github/workflows/check-pr-title.yml:1]: Q: Does User must find a gitmoji from https://gitmoji.dev/?

Comment (hwbrzzl) [reply .github/workflows/check-pr-title.yml:1]: I think it's a bit complex, we need to open the website and copy one gitmoji to here, I suggest we can check the title that starts with Feat/Chore/Fix if needed.

Comment (DevHaoZi) [reply .github/workflows/check-pr-title.yml:1]: > I think it's a bit complex, we need to open the website and copy one gitmoji to here, I suggest we can check the title that starts with Feat/Chore/Fix if needed.

ok, let me change it.

**Pull request merged**